pkg/node/node_connectivity_checker.go

```diff
@@ -338,14 +338,21 @@ func (s *Service) populateTargetsInCache(array *array.PowerStoreArray) {
 			log.Errorf("couldn't get targets from array: %s", err.Error())
 			return
 		}
-
+		var ipAddress string
 		var iscsiTargets []goiscsi.ISCSITarget
 		for _, address := range infoList {
 			// first check if this portal is reachable from this machine or not
 			if ReachableEndPoint(address.Portal) {
+				ipAddressList := strings.Split(address.Portal, ":")
```

Comment:
Same here for use of colons. You could encapsulate this in a common utility function.